packages/app-desktop/gui/NoteEditor/utils/contextMenuUtils.test.ts

```diff
@@ -0,0 +1,35 @@
+import { svgDimensions } from './contextMenuUtils';
+
+describe('contextMenuUtils',()=>{
+	test('should return the correct dimensions for svg with space seperated viewBox values',()=>{
+		const svg = '<svg viewBox="0 0 45.9375 60" "></svg> ';
+		const [width,height] = svgDimensions(svg);
+		expect(width).toBe(45);
+		expect(height).toBe(60);
+	});
+	test('should return the correct dimensions for svg with comma seperated viewBox values',()=>{
+		const svg = '<svg viewBox="0,0,45.9375,60" "></svg> ';
+		const [width,height] = svgDimensions(svg);
+		expect(width).toBe(45);
+		expect(height).toBe(60);
+	});
+	test('should return the correct dimensions for svg with comma + space seperated viewBox values',()=>{
+		const svg = '<svg viewBox="0, 0, 45.9375, 60" "></svg> ';
+		const [width,height] = svgDimensions(svg);
+		expect(width).toBe(45);
+		expect(height).toBe(60);
+	});
+	test('should return undefined for svg without viewBox',()=>{
+		const svg = '<svg></svg> ';
+		const [width,height] = svgDimensions(svg);
+		expect(width).toBe(undefined);
+		expect(height).toBe(undefined);
+	});
+	test('should return undefined for svg with NaN width and height',()=>{
+		const svg = '<svg viewBox="0 0 troll svg" "></svg> ';
+		const [width,height] = svgDimensions(svg);
+		expect(width).toBe(undefined);
+		expect(height).toBe(undefined);
+	});
+
```

Comment:
Please also consider the following cases:
- `viewBox` has the form `viewBox='1,2,3,4'` or `viewBox   = "1,2,3,4"` instead of `viewBox="1,2,3,4"`
- `viewBox` has the form
```
viewBox='
	1,
	2,
	3 ,
	4 ,
'
```